Makefile

```diff
@@ -11,7 +11,7 @@ GAME ?= standard
 
 CPLUS = g++
 CC = gcc
-CFLAGS = -g -I. -I.. -Wall -Werror -std=c++11
+CFLAGS = -g -I. -I.. -Wall -Werror -std=c++20
```

Comment:
The new unit test framework requires C++20 support.   All of our build compilers support it.